docs/runbooks/KubeVirtDeprecatedAPIsRequested.md

```diff
@@ -0,0 +1,29 @@
+# KubeVirtDeprecatedAPIsRequested
+<!-- Edited by fmatouschek, May 2023-->
+
+## Meaning
+
+This alert fires when a deprecated KubeVirt API was requested.
+
+## Impact
+
+Usage of deprecated APIs is not recommended because they will be removed in a future release.
+
+## Diagnosis
+
+Check the actual deprecated API usage by using a PromQL query:
+
+```
+kubevirt_api_request_deprecated_total
+```
+
+See the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/querying/basics/) for more information.
+
+## Mitigation
+
+Make sure to only use a supported version when making requests to the API.
+
+Some requests to deprecated APIs are made by KubeVirt components themselves (e.g VirtualMachineInstancePresets).
+These alerts cannot be mitigated because the requests are still necessary to serve the deprecated API.
+They are harmless and will be resolved when the deprecated API is removed in a future release of KubeVirt.
```

Comment:
@0xFelix  Is there a way to identify who initiated the request and only show the ones not made by the KubeVirt components?

Also, In every runbook we also add the following:
```
<!--DS: If you cannot resolve the issue, log in to the link:https://access.redhat.com[Customer Portal] and open a support case, attaching the artifacts gathered during the Diagnosis procedure.-->
<!--USstart-->
See the [HCO cluster configuration documentation](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/docs/cluster-configuration.md#enablecommonbootimageimport-feature-gate) for more information.

If you cannot resolve the issue, see the following resources:

- [OKD Help](https://www.okd.io/help/)
- [#virtualization Slack channel](https://kubernetes.slack.com/channels/virtualization)
<!--USend-->
```

Reply:
Updated the PR to include this block.